src/Adapter/index.js

```diff
@@ -6,7 +6,19 @@ const options = {
   valueEncoding: 'json',
 };
 
-const { union } = Gun.HAM;
+const union = function union(vertex, node, opt){
+	if(!node || !node._){ return }
+	vertex = vertex || Gun.state.to(node);
+	if(!vertex || !vertex._){ return }
+	opt = Gun.num.is(opt)? {machine: opt} : {machine: Gun.state()};
+	opt.union = Gun.obj.copy(vertex); // Slow performance.
+	if(!Gun.node.is(node, function(val, key){
+		var HAM = Gun.HAM(opt.machine, Gun.state.is(node, key), Gun.state.is(vertex, key, true), val, vertex[key]);
+		if(!HAM.incoming){ return }
+		Gun.state.to(node, key, opt.union);
+	})){ return }
+	return opt.union;
+}
```

Comment:
I suspect this is why [eslint freaked out](https://travis-ci.org/PsychoLlama/gun-level/jobs/252397528). I'm glad you added this into gun-level, now future breaking changes in gun core shouldn't affect this.

I would not object if you put `/* eslint-disable */` before the union code block and ended it with `/* eslint-enable */`. That should fix the rest of the build.